org.OpenT2T.Sample.SuperPopular.Thermostat/Wink Thermostat/js/thingTranslator.js

```diff
@@ -0,0 +1,146 @@
+'use strict';
+
+var https = require('https');
+var wh = require("org.OpenT2T.Sample.SuperPopular.Helpers/js/index.js"); //change this to your local
```

Comment:
This should require the helper via an npm dependency: `require('opent2t-translator-helper-wink')`
Note you can `npm install` a package from a local path even before it is published to npm.